app/classes/api/name_api.rb

```diff
@@ -239,12 +239,12 @@ def parse_set_synonymy!
     end
 
     # Disable cop because there's no reasonable way to avoid the offense
-    # rubocop:disable CyclomaticComplexity
+    # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
RuboCop `disable`/`enable` within our source code must now include the cop's "department" (namespace).